tests/bases/test_ddp.py

```diff
@@ -140,6 +140,7 @@ def compute(self):
 
     metric = DummyCatMetric()
     metric.persistent(True)
+    _ = metric.state_dict()
```

Comment:
is this a requirement for things to work?

Reply:
No, but the error is reproduced whenever `state_dict` is called before update is called. So adding this will make sure the error is fixed